jerry-core/api/jerry.c

```diff
@@ -2050,6 +2050,42 @@ jerry_get_boolean_value (const jerry_value_t value) /**< api value */
   return ecma_is_value_true (value);
 } /* jerry_get_boolean_value */
 
+/**
+ * Check if the specified value is true.
+ *
+ * @return true or false.
+ */
+
+bool
+jerry_is_value_true (const jerry_value_t value)
+{
+  jerry_assert_api_available ();
+
+  if (jerry_value_is_boolean(value))
+  {
+    return jerry_get_boolean_value(value) == true ? true : false;
+  }
+  return false;
+}/* jerry_is_value_true */
+
+/**
+ * Check if the specified value is false.
+ *
+ * @return true or false.
+ */
+
+bool
+jerry_is_value_false (const jerry_value_t value)
+{
+  jerry_assert_api_available ();
+
+  if (jerry_value_is_boolean(value))
+  {
+    return jerry_get_boolean_value(value) == false ? true : false;
+  }
+  return false;
```

Comment:
It can be a oneliner:
return jerry_value_is_boolean(value) && !jerry_get_boolean_value(value);